include/defines.h

```diff
@@ -275,9 +275,9 @@ enum KART_AI_BEHAVIOURS {
     BEHAVIOUR_NONE = 0,
     BEHAVIOUR_1,
     BEHAVIOUR_HOP,
-    BEHAVIOUR_3,
-    BEHAVIOUR_4,
-    BEHAVIOUR_5,
+    BEHAVIOUR_GO_CENTER,
```

Comment:
BEHAVIOUR_DRIVE_CENTER ? DRIVE_CENTERLINE ?

Reply:
he will try to go to the center